### PR TITLE
Fix #8 Added eslint

### LINKED ISSUE #8: Use Eslint
In combination with #7, let's use [eslint](https://eslint.org/docs/user-guide/getting-started) to automatically find and fix issues in our code.

### PR DESCRIPTION
**Pull request by Fatehsandhu**
Fixing #8  Adding [eslint](https://github.com/eslint/eslint) to our code.

1. Eslint is a tool for identifying and reporting on patterns in our .js files.
2. I am installing eslint globally and then we can use it all over our code.
3. I am using the standard configuration to check our JS code.
4. We can add more configurations later depending on our needs.

You can test this by running the following commands.
```bash
npm install -g eslint
//then run this on a file to test it
eslint nameOfFile.js
```
Replaces #66

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    "extends": "standard"
```
Comment (Fatehsandhu) [inline .eslintrc.js]: Change to recommend rather than standard.

File: package.json [hunk 2]
```diff
@@ -23,6 +25,12 @@
         "filer": "humphd/filer#9p-filer"
     },
     "devDependencies": {
+        "eslint": "^4.19.1",
+        "eslint-config-standard": "^11.0.0",
```
Comment (Fatehsandhu) [inline package.json]: Remove this since we will remove standard

File: package.json [hunk 3]
```diff
@@ -23,6 +25,12 @@
...
+        "eslint-plugin-import": "^2.12.0",
+        "eslint-plugin-node": "^6.0.1",
+        "eslint-plugin-promise": "^3.8.0",
+        "eslint-plugin-standard": "^3.1.0",
```
Comment (Fatehsandhu) [inline package.json]: Remove this as well

**Comment by dsych** [comment]
@Fatehsandhu we should also integrate prettier with eslint to avoid conflict between the two. [This](https://github.com/prettier/eslint-plugin-prettier) should be useful.

**Pull request merged**